docs/object-attribute-reference.md

```diff
@@ -0,0 +1,15 @@
+# Object Attribute Reference
+## List all preset object attributes
+
+---
+
+| Attribute         | Type      | Value | Description |
+|-------------------|-----------|-------|-------------|
+| `xp`              | function  | A valid function to calculate experience points, such as `exerciceExpCalculation`, `examExpCalculation` or `raidExpCalculation` | This attribute define via a function the amount of exeperience points that an object will give. |
```

Comment:
Toutes les fonctions doivent etre valide, ca me semble redondant de specifier `a valid function` meme le fait que c'est une fonction on le sais juste avant avec sont type

Reply:
pareil pour les descriptions, on a pas besoin de dire `this attribute` a chaques fois, ni que c'est via une fonction pour une 3eme fois:

> the amount of experience points that an object will give.

Short & sweet